extmod/modtls_mbedtls.c

```diff
@@ -267,6 +286,9 @@ STATIC void ssl_context_attr(mp_obj_t self_in, qstr attr, mp_obj_t *dest) {
             self->authmode = mp_obj_get_int(dest[1]);
             dest[0] = MP_OBJ_NULL;
             mbedtls_ssl_conf_authmode(&self->conf, self->authmode);
+        } else if (attr == MP_QSTR_verify_callback) {
+            dest[0] = MP_OBJ_NULL;
+            ssl_context_set_verify_callback(self, dest[1]);
```

Comment:
I don't think this needs to be a separate helper function (to set the callback).  It can simply be `self->handler = dest[1]`.